src/base/bittorrent/torrenthandle.h

```diff
@@ -331,7 +331,7 @@ namespace BitTorrent
         void setName(const QString &name);
         void setSequentialDownload(bool b);
         void toggleSequentialDownload();
-        void setFirstLastPiecePriority(bool b);
+        void setFirstLastPiecePriority(const bool enabled);
```

Comment:
I can understand the necessity of const params in method implementation, but it is unacceptable in the interface! It "confuses the whole outside world".
In method definition it means "you can't change the variable" and it's ok.
But what it means in the method interface (i.e declaration)? Argument couldn't be changed by the method? This is absurd! The argument passed by value cannot be changed by the method anyway.

Reply:
Then in the cpp file it will have `const` while the header won't, is it ok for you?

Reply:
Yes, it is.

Reply:
ok, reverted.